scripts/translate/de_DE.po

```diff
@@ -237,39 +237,39 @@ msgstr "Standard-Spiel"
 
 #: guichan.lua:1127
 msgid "~!Cancel"
-msgstr "Abbrechen (~!c)"
+msgstr "Abbru~!ch"
 
 #: guichan.lua:1150
 msgid "~red~Red"
-msgstr "~red~Rot"
+msgstr "~rot~Rot"
```

Comment:
I think that `~color~` has special meaning and should not be translated (purpose is to have text "Red" in red).
Apply to some other places too.